pmd-plsql/etc/grammar/PldocAST.jjt

```diff
@@ -4836,7 +4875,12 @@ TOKEN [IGNORE_CASE]:
 <FOREACHROW: "FOR EACH ROW"> |
 <REFERENCING: "REFERENCING"> |
 <OLD: "OLD"> |
-<PARENT: "PARENT">
+<PARENT: "PARENT"> |
```

Comment:
I tried to avoid adding more tokens for unreserved keywords in the past and used instead `KEYWORD("XYZ")`. The reason is, that the unreserved keywords are available for identifiers as well and adding more tokens to the KEYWORD_UNRESERVED production at some point generates a switch statement, that is too big for the java compiler... since your change compiles, we didn't hit this problem yet...